Source/JavaScriptCore/runtime/JSGlobalObjectFunctions.cpp

```diff
@@ -1085,4 +1085,9 @@ JSC_DEFINE_HOST_FUNCTION(globalFuncToLength, (JSGlobalObject* globalObject, Call
     return JSValue::encode(jsNumber(argument.toLength(globalObject)));
 }
 
+JSC_DEFINE_HOST_FUNCTION(globalFuncSpeciesGetter, (JSGlobalObject*, CallFrame* callFrame))
+{
+    return JSValue::encode(callFrame->thisValue());
```

Comment:
While here we do need to call `.toThis(globalObject, ECMAMode::strict())`.

A test with `JSGlobalObject` being leaked:

```
var x = Object.getOwnPropertyDescriptor(Array, Symbol.species).get;

print(x());
```

Reply:
Done.